src/tests/nativeaot/SmokeTests/Preinitialization/Preinitialization.cs

```diff
@@ -1310,15 +1311,17 @@ public static void Run()
 
 class TestIndirectLoads
 {
-    static unsafe U Read<T, U>(T val) where T : unmanaged where U : unmanaged
```

Comment:
Roslyn generates this as `new()` constraint which is very unfortunate because it requires dataflow.